includes/NSFW.h

```diff
@@ -58,9 +58,41 @@ class NSFW : public Napi::ObjectWrap<NSFW> {
 
     Napi::Value Stop(const Napi::CallbackInfo &info);
 
+    class PauseWorker: public Napi::AsyncWorker {
+      public:
+        PauseWorker(Napi::Env env, NSFW *nsfw);
+        void Execute();
+        void OnOK();
+        Napi::Promise RunJob();
+
+      private:
+        Napi::Promise::Deferred mDeferred;
+        bool mDidPauseEvents;
```

Comment:
Oops, I missed this one :). We should make this an atomic as well.

Reply:
ah of course